src/coreclr/src/gc/gc.cpp

```diff
@@ -13081,14 +14048,18 @@ size_t gc_heap::get_full_compact_gc_count()
 // DTREVIEW - we should check this in dt_low_ephemeral_space_p
 // as well.
 inline
-BOOL gc_heap::short_on_end_of_seg (heap_segment* seg, int align_const)
+BOOL gc_heap::short_on_end_of_seg (int gen_number, heap_segment* seg)
```

Comment:
Nit: Is gen_number argument used anywhere?

Reply:
it's not; I was using it but then did a code clean up and didn't notice I wasn't using it anymore. I'll make a note of this (will either do it in this PR or with cleanup PR).

Reply:
Ok.